migration/migration.go

```diff
@@ -299,3 +304,33 @@ func (m *changeTrieNodeEncoding) Migrate(txn db.Transaction) error {
 	}
 	return iterator.Close()
 }
+
+// calculateBlockCommitments calculates the txn and event commitments for each block and stores them separately
+func calculateBlockCommitments(txn db.Transaction, network utils.Network) error {
+	var txnLock sync.RWMutex
+	workerPool := pool.New().WithErrors().WithMaxGoroutines(runtime.GOMAXPROCS(0))
+	blockNumber := 0
+
+	for {
```

Comment:
```suggestion
	for blockNumber := 0; ; blockNumber++ {
```
And then remove `blockNumber++` at the end of the loop.

Reply:
Thanks:)